app/uk/gov/hmrc/cgtpropertydisposalsfrontend/controllers/returns/yeartodatelliability/YearToDateLiabilityController.scala

```diff
@@ -822,7 +822,13 @@ class YearToDateLiabilityController @Inject() (
                 draftReturn: DraftReturn
               ) =>
             handleNonCalculatedEstimatedDetailsSubmit(
-              nonCalculatedAnswers,
+              if (fillingOutReturn.isFurtherReturn.contains(true))
```

Comment:
the changing of the answers is done within `handleNonCalculatedEstimatedDetailsSubmit` - you don't need to change it at the point of calling the function